contrib/platform/src/com/sun/jna/platform/win32/Advapi32Util.java

```diff
@@ -3075,7 +3047,7 @@ public static int fileEncryptionStatus(File file) {
      *         TRUE to disable encryption. FALSE to enable it.
      */
     public static void disableEncryption(File directory, boolean disable) {
-        String dirPath = directory.getAbsolutePath();
+        WString dirPath = new WString(directory.getAbsolutePath());
```

Comment:
Why is this one explicitly a `WString`?

Reply:
According to MSDN, EncryptionDisable only accepts LPCWSTR, so this one needs to be forced to wchar_t* even on ASNI API?